docs/migration/migrate_7_8.asciidoc

```diff
@@ -0,0 +1,523 @@
+search.aggs.AggConfigs;
+search.aggs.aggGroupNamesMap, // renamed to AggGroupLabels
+search.aggs.aggTypeFilters,
+search.aggs.convertDateRangeToString,
+search.aggs.convertIPRangeToString,
+```
+
+*via {pull}64719[#64719]*
+====
+
+[[breaking_78_update_default_path]]
+.Applications are now allowed to define and update a `defaultPath`.
+[%collapsible]
+====
+
+{kib} platform applications can now define and update the `defaultPath`
+to use when navigating to them from another application or from the navigation bar.
+
+
+```ts
+core.application.register({
+    id: 'my-app',
+    // ...
+    defaultPath: '/some-path',
+})
+```
+
+```ts
+const appUpdater = new BehaviorSubject<AppUpdater>(() => ({}));
+core.application.register({
+    id: 'my-app',
+    // ...
+    updater$: appUpdater,
+})
+
+// later
+appUpdater.next(() => ({ defaultPath: '/some-updated-path' }));
+```
+
+*via {pull}64498[#64498]*
+====
+
+[[breaking_78_cache_busting]]
+.Static assets are now served under a release-specific URL.
+[%collapsible]
+====
+
+{kib} static assets are now served under a release-specific URL
+with long-term caching headers `Cache-Control: max-age=31536000`.
+
+Before:
+
+http://localhost:5601/bundles/plugin/dashboard/dashboard.plugin.js
+
+After:
+
+http://localhost:5601/bundles/8467/plugin/dashboard/dashboard.plugin.js
+
+*via {pull}64414[#64414]*
+====
+
+[[breaking_78_add_example_plugins]]
+.Example plugins are now allowed in X-Pack.
+[%collapsible]
+====
+
+{kib} developers can now create example plugins in
+X-Pack&mdash;create your plugin in `/x-pack/examples` folder and start {kib} with:
+
+```ts
+yarn start --run-examples
+```
+
+*via {pull}63823[#63823]*
+====
+
+[[breaking_78_improves_drilldows]]
+.`action.getHref()` has improvements for drilldowns.
+[%collapsible]
+====
+
+`getHref` on `Action` interfaces in the uiActions plugin is now async.
+`getHref` is now used only to support right click behavior.
+`execute()` takes control on regular click.
+
+*via {pull}63228[#63228]*
+====
+
+[[breaking_78_scopedhistory]]
+.State syncing utils now support `ScopedHistory`.
+[%collapsible]
+====
+
+State syncing utils now seamlessly support the platform's `ScopedHistory`.
+
+*via {pull}62761[#62761]*
+
+====
+
+[[breaking_78_tsvb]]
+.Configuration properties were removed from TSVB.
+[%collapsible]
+====
+
+When the TSVB visualization was added to {kib},
+two configuration properties were declared: `chartResolution` and `minimumBucketSize`.
+No one used these properties, and
+an implementation has not been added.
+The `chartResolution` and `minimumBucketSize` are now marked as deprecated configuration
+properties for TSVB.
+
+*via {pull}62543[#62543]*
+
+====
+
+[[breaking_fixes_back_button]]
+.Panels were removed from the URL in dashboard view mode.
+[%collapsible]
+====
+
+In dashboard in view mode, `.panels` are no longer synced with the URL.
+This fixes the Back button when navigating between dashboards using drilldowns.
+
+*via {pull}62415[#62415]*
+====
```

Comment:
My understanding is that it is a breaking change for, both, users as well as plugin developers. But the breaking change is in the Canvas app, so I ping them for commnet: @timductive @clintandrewhall  @crob611

Reply:
To me it feels like (if it is a breaking change) it is, both, user and plugin developer facing breaking change. The change is basically that in a long Kibana URL like

```
http://kibana:5620/dashboard/xxx#...&panels={...}
```

The `panels={ ... }` variable is no longer present, like this

```
http://kibana:5620/dashboard/xxx#...
```

This change is technically in KibanaApp team's codebase, so I'll ping them to comment on how they see it: @timroes @ThomThomson @ppisljar

Reply:
This seems to me like more of a user facing change because it affects view mode. I agree that it does apply to both plugin devs and users, but moving it up to the user facing section will give it a little bit more weight visually.